src/runtime/arc.ts

```diff
@@ -547,22 +511,12 @@ export class Arc implements ArcInterface {
         assert(storageKey, `couldn't find storage key for handle '${recipeHandle}'`);
         let type = recipeHandle.type.resolvedType();
         assert(type.isResolved());
-        if (typeof storageKey === 'string') {
-          const store = await this.storageProviderFactory.connect(recipeHandle.id, type, storageKey);
-          // Note that at one stage this assertion ensured that recipes couldn't "use" handles which weren't
```

Comment:
do you want to keep this comment? is it still relevant?

Reply:
I don't think so; the lack of issues since making that change indicate that the "Hopefully" is correct. There's no point continuing to point out the incorrect thing we used to do.